tools/clang/lib/Format/TokenAnnotator.cpp

```diff
@@ -2260,7 +2260,7 @@ bool TokenAnnotator::canBreakBefore(const AnnotatedLine &Line,
        Left.getPrecedence() == prec::Assignment))
     return true;
   return Left.isOneOf(tok::comma, tok::coloncolon, tok::semi, tok::l_brace,
-                      tok::kw_class, tok::kw_struct) ||
+                      tok::kw_class, tok::kw_struct, tok::kw_union) ||
```

Comment:
Same as above. C++ doesn't need this, so HLSL shouldn't either.

Reply:
Done